### PR TITLE
solution added

### PR DESCRIPTION
**Pull request by dariiabutorina**

### CODE REVIEW

File: src/main/java/core/basesyntax/Plane.java [hunk 1]
```diff
@@ -7,5 +7,75 @@
...
 
 public class Plane {
+    private String model;
+    private String typeBySize;
+    private String typeByLanding;
+    private String typeByPurpose;
+    private String companyOwner;
 
+    public Plane(PlaneBuilder builder) {
+        this.model = builder.model;
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Plane.java]:
```suggestion
        model = builder.model;
```

File: src/main/java/core/basesyntax/Plane.java [hunk 2]
```diff
@@ -7,5 +7,75 @@
...
+        this.typeBySize = builder.typeBySize;
+        this.typeByLanding = builder.typeByLanding;
+        this.typeByPurpose = builder.typeByPurpose;
+        this.companyOwner = builder.companyOwner;
+    }
+
+    public String getModel() {
+        return this.model;
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Plane.java]:
```suggestion
        return model;
```

File: src/main/java/core/basesyntax/Plane.java [hunk 3]
```diff
@@ -7,5 +7,75 @@
  */
 
 public class Plane {
+    private String model;
+    private String typeBySize;
+    private String typeByLanding;
+    private String typeByPurpose;
+    private String companyOwner;
 
+    public Plane(PlaneBuilder builder) {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/Plane.java]: we should be able to call this constructor from inside this class only
```suggestion
   private Plane(PlaneBuilder builder) {
```

File: src/main/java/core/basesyntax/Plane.java [hunk 4]
```diff
@@ -7,5 +7,75 @@
...
+        return companyOwner;
+    }
+
+    public static class PlaneBuilder {
+
+        public String model;
+        public String typeBySize;
+        public String typeByLanding;
+        public String typeByPurpose;
+        public String companyOwner;
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/Plane.java]: don't forget about field encapsulation

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request closed without merging**